legacy_tester/parsed_json_test_definitions.py

```diff
@@ -20,3 +20,17 @@ def count_of_tonal_remarks_containing_test_string(published_json, test_payload):
                                                                                          unit_payload['test_string']))
             return False
     return True
+
+
+def subtypes_should_not_have_string(published_json, test_payload):
+    """Test for issue #76
+    This is to test if the parser has successfully ignored <td> blocks that don't
+    contain attributes pertaining to platform sub type <td>
+    """
+    for test_string in test_payload:
```

Comment:
hmm, it looks like `test_payload` for `subtypes_should_not_have_string` doesn't have a `test_string` named property. It looks like it just has the string to match.

Reply:
The data type of `test_payload` is purely contextual. In this case, it contains a list of string that should not be present in the extracted sub types just like `Britain1` in `phase_one_regions_parse_not_included_countries`